portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var firstSlide = document.getElementById("firstSlide");
+  if (n > slides.length + 1) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length + 1}
+  for (i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+  }
+  if (slideIndex == 1){
```

Comment:
Be aware of JS type coercion. Use '===' and '!==' instead of '==' and '!=' except when it's desirable.

https://google.github.io/styleguide/jsguide.html#features-equality-checks

Reply:
done